### ISSUE TITLE
Re-check for sand with area mining

### ISSUE THREAD

**Issue by SylveonBottle** [cabaletta/baritone#398]
After an area mine, if the selected area contained any said, bariton should do one final re-check once it's complete to make sure that no sand fell.

https://gfycat.com/SafeCarefreeBuzzard

**Comment by SylveonBottle** [comment]
I believe this was somewhat """fixed""" by making baritone no longer dig out falling blocks?

**Comment by leijurv** [comment]
Not quite, while the falling check was added for each individual _movement_ (particularly useful during traverse), it's not included in builderprocess's calculation of which positions contain invalid blocks. So this is still an issue as it stands.
